meticulous/experiment.py

```diff
@@ -277,7 +279,7 @@ def _set_experiments_directory(self, experiments_directory):
         except FileNotFoundError as e:
             print("Creating local .gitignore")
             ignored = False
-        if not ignored:
+        if not ignored and not os.path.relpath(self.experiments_directory, self.repo_directory).startswith(".."):
```

Comment:
Can you explain why we're doing this? Maybe just leave a comment in the code as well?

Reply:
Sure. I had an issue when I needed to store my results outside of the repository (think of larger models that need to go onto a local disk instead of a slower network filesystem).

However, adding a file-path that's outside the repo to the .gitignore makes no sense, so I inserted this.

I now see that in the readme you say that the experiments folder must be inside of the repo. I don't really see why?

Reply:
You are right, there is no reason for that. And I'm guessing you the relpath followed startswith is to determine if the the experiments directory is outside repo directory. I think if you could leave a single line comment explaining it that would be great.